package.json

```diff
@@ -280,6 +280,86 @@
                   "type": "string"
                 },
                 "default": []
+              },
+              "pipeTransport": {
+                "type": "object",
+                "description": "Describes options to launch a program through a pipe transport.",
```

Comment:
How about:When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).